vnext/include/ReactUWP/IReactInstance.h

```diff
@@ -31,6 +31,9 @@ struct ReactInstanceSettings
   bool UseDirectDebugger{ false };
   bool UseJsi { true };
   bool EnableJITCompilation { true };
+  bool EnableByteCodeCacheing { false };
+
+  std::string ByteCodeFileUri;
```

Comment:
>ByteCodeFileUri [](start = 14, length = 15)

Looks ok for now .. But, ReactUWP might require loading multiple bundles at some point in time in future ..

Reply:
Having some kind of naming convention to the bytecode files based on the source url is one option ..

---
In reply to: [295954056](https://github.com/microsoft/react-native-windows/pull/2653#discussion_r295954056) [](ancestors = 295954056)